frontend/models/product.ts

```diff
@@ -7,6 +7,7 @@ import {
    MoneyV2,
    CurrencyCode,
 } from '@lib/shopify-storefront-sdk';
+import { z } from 'zod';
```

Comment:
Question: I see that [`zod recommends`](https://github.com/colinhacks/zod#requirements) using strict mode typescript, but I don't think we're doing that in `frontend/tsconfig.json`. Do we care? Seems like things are working, so maybe it doesn't matter.